doc/releases/mimic.rst

```diff
@@ -245,7 +397,7 @@ Notable Changes
 
 
 Other Notable Changes
-~~~~~~~~~~~~~~~~~~~~~
+---------------------
```

Comment:
This doesn't seem to make any difference in the rendered doc. I found the same - *no effect* in other docs too.